ArduPlane/mode.h

```diff
@@ -225,9 +225,22 @@ class ModeGuided : public Mode
     // handle a guided target request from GCS
     bool handle_guided_request(Location target_loc) override;
 
+    void set_guided_radius(const float radius) {
+        // convert radius value
+        // from: -1 = use default param WP_LOITER_RAD, 0 = ignore (no-change), >0 radius
+        // to: <= 0 use default param WP_LOITER_RAD, >0 radius_m
+        // we constrain to uint16_t because that is what update_loiter() is currently limited to
+        if (!is_zero(radius)) {
+            active_radius_m = (radius < 0) ? 0 : MIN(radius,UINT16_MAX);
```

Comment:
set to default radius is <= 0, so it doesn't persist

Reply:
fixed